cluster/aws/util.sh

```diff
@@ -1511,10 +1511,12 @@ function ssh-to-node {
   fi
 
   for try in $(seq 1 5); do
-    if ssh -oLogLevel=quiet -oStrictHostKeyChecking=no -i "${AWS_SSH_KEY}" ${SSH_USER}@${ip} "${cmd}"; then
+    if ssh -oLogLevel=quiet -oConnectTimeout=30 -oStrictHostKeyChecking=no -i "${AWS_SSH_KEY}" ${SSH_USER}@${ip} "echo test > /dev/null"; then
```

Comment:
Could be. `LogLevel` is already repeated all over the place, and `StrictHostKeyChecking` is used inconsistently. It might be reasonable to put it in `cluster/common.sh`.